src/renderer/components/TransactionView.tsx

```diff
@@ -0,0 +1,110 @@
+import { faPencil, faSignature } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import * as sol from '@solana/web3.js';
+import { netToURL, prettifyPubkey } from 'common/strings';
+import { useEffect, useState } from 'react';
+import { Badge, ListGroup, ListGroupItem } from 'react-bootstrap';
+import { Net } from 'types/types';
+
+type InstructionAccountData = {
+  instruction: sol.ParsedInstruction;
+  accountKeys: sol.ParsedMessageAccount | undefined;
+};
+
+function TransactionView(props: { net: Net }) {
+  const { net } = props;
+  const [latestProcessedSlot, setLatestProcessedSlot] = useState(0);
+  const [renderedParsedTransactions, setRenderedParsedTransactions] = useState<
+    (sol.ParsedTransactionWithMeta | null)[][]
+  >([]);
+  useEffect(() => {
+    const blockInterval = setInterval(async () => {
+      const solConn = new sol.Connection(netToURL(net));
+      const blockProduction = await solConn.getBlockProduction();
+      const blocks = await solConn.getBlocks(
```

Comment:
I was going to ask for the subscription code to be separated out into `../data/transactions.ts`, but actually - this feels small and targeted, and who needs premature extraction :)